lbplanner/services/user/delete_user.php

```diff
@@ -25,48 +25,49 @@
 use local_lbplanner\helpers\plan_helper;
 use local_lbplanner\helpers\course_helper;
 use local_lbplanner\helpers\notifications_helper;
+use moodle_exception;
 
 /**
  * Removes all user data stored by the lbplanner app
  */
 class user_delete_user extends external_api {
-    public static function delete_user_parameters() {
-        return new external_function_parameters(array(
-            'userid' => new external_value(
-                PARAM_INT,
-                'The id of the user to get the data for',
-                VALUE_REQUIRED,
-                null,
-                NULL_NOT_ALLOWED
-            ),
-        ));
+    public static function delete_user_parameters(): external_function_parameters {
+        return new external_function_parameters(array());
```

Comment:
i'd still add the userid as an optional parameter, so admins can delete a user if necessary

Reply:
only admins or managers aswell?

Reply:
hmm i'm not sure, i think admins, but what do you think?

Reply:
i currently added it only for admins. Because managers shouldnt be allowed to have that i high of an impact to delete the acc

Reply:
oki